frontend/src/employee-frontend/components/unit/TabCalendar.tsx

```diff
@@ -128,10 +128,7 @@ export default React.memo(function TabCalendar() {
           <Gap size="s" />
         </>
       ) : null}
-
-      {unitInformation.permittedActions.has('READ_CHILD_ATTENDANCES') ? (
-        <Calendar unitId={unitId} unitInformation={unitInformation} />
-      ) : null}
+      <Calendar unitId={unitId} unitInformation={unitInformation} />
```

Comment:
Now the calendar (or rather it's error segment) is shown for users that can't read all the data, eg. users with `DIRECTOR` role.